asyncapi/github/service_types.bal

```diff
@@ -93,14 +97,14 @@ public type MilestoneService service object {
     remote function onOpened(MilestoneEvent payload) returns error?;
 };
 
-# Actions related to push events. This will be triggered when one or 
-# more commits are pushed to a repository branch or tag.  
+# Triggers when a push event is received.
+# Available action: onPush
 public type PushService service object {
     remote function onPush(PushEvent payload) returns error?;
 };
 
-# Actions related to project cards. The action can be one of 
-# created, edited, moved, converted or deleted.
+# Triggers when a new event related to project cards is received.
```

Comment:
Suggestion `Triggers when a new event related to ProjectCard event is received.`